python/tvm/relay/frontend/onnx.py

```diff
@@ -1530,6 +1532,14 @@ def _activation_needs_beta(cls, activation):
 
     @classmethod
     def _impl_v7(cls, inputs, attr, params):
+        if cls.name == 'LSTM':
```

Comment:
here I think it'd be cleaner to just implement the `@classmethod` in each subclass, no?

Reply:
you're right thats a nicer organization. I've updated the structure accordingly.